doc/nrf/ug_nrf52_gs.rst

```diff
@@ -0,0 +1,270 @@
+.. _ug_nrf52_gs:
+
+Getting started with nRF52 Series
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section gets you started with your nRF52 Series :term:`Development Kit (DK)` using the |NCS|.
+It tells you how to install the :ref:`peripheral_uart` sample and perform a quick test of your DK.
+
+If you have already set up your nRF52 Series DK and want to learn more, see the following documentation:
+
+* :ref:`ug_nrf52` for more advanced topics related to the nRF52 Series if you are already familiar with the |NCS|.
+* The :ref:`introductory documentation <getting_started>` for more information on the |NCS| and the development environment.
+
+.. _nrf52_gs_requirements:
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer and mobile device both have one of the supported operating systems.
+
+Hardware
+========
+
+* One of the following nRF52 Series development kits:
+
+  * nRF52840 DK
+  * nRF52833 DK
+  * nRF52 DK
+
+* Micro-USB 2.0 cable
+
+Software
+========
+
+On your computer, one of the following operating systems:
+
+* Microsoft Windows 10
+* macOS X, latest version
+* Ubuntu Linux, latest Long Term Support (LTS) version
+
+On your mobile device, one of the following operating systems:
+
+* Android
+* iOS
+
+.. _nrf52_gs_installing_software:
+
+Installing the required software
+********************************
+
+On your computer, install `nRF Connect for Desktop`_.
+After installing and starting the application, install the Programmer app.
```

Comment:
```suggestion
After installing and starting the application, install the Programmer application.
```

Reply:
Or is just referred to as "Programmer app"? I see this on line 81 as well.

Reply:
I believe it's generally considered an app since it's inside the nRF Connect for Desktop application. At least, that's how I've referred to it before.